src/pybind/mgr/dashboard/frontend/src/app/shared/api/rgw-user.service.ts

```diff
@@ -131,10 +131,11 @@ export class RgwUserService {
    * @return {Observable<boolean>}
    */
   exists(uid: string): Observable<boolean> {
-    return this.enumerate().pipe(
-      mergeMap((resp: string[]) => {
-        const index = _.indexOf(resp, uid);
-        return observableOf(-1 !== index);
+    return this.get(uid).pipe(
+      mapTo(true),
+      catchError((error) => {
```

Comment:
If possibe please define the type (HttpEvent????), so TypeScript is aware of the type.
```suggestion
      catchError((error: XXXX) => {
```

Reply:
It was actually an Event I think. @votdev